src/components/hr/hr.pw.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+import {
+  Default,
+  DifferentSpacing,
+  EnablingAdaptiveBehaviour,
+  InsideForm,
+  InsideFormInlineLabels,
+} from "../hr/components.test-pw";
+import hrComponent from "../../../playwright/components/hr/index";
+import HrComponent from "./hr.component";
+
+test.describe("check props for Hr component", () => {
+  test("verify Hr component is displayed properly", async ({ mount, page }) => {
+    await mount(<HrComponent />);
+
+    await expect(hrComponent(page)).toHaveCSS("margin-top", "24px");
+    await expect(hrComponent(page)).toHaveCSS("margin-bottom", "24px");
+  });
+
+  [
+    [799, 80, 320],
+    [800, 80, 320],
+    [801, 0, 0],
+  ].forEach(([breakpoint, leftMargin, rightMargin]) => {
+    test.use({ viewport: { width: 800, height: 300 } });
+
+    test(`verify Hr component adaptiveMaxBreakpoint set to ${breakpoint} prop sets left and right margins to 0px when larger than viewport`, async ({
+      mount,
+      page,
+    }) => {
+      await await mount(
+        <HrComponent
+          mb={7}
+          mt={7}
+          ml="10%"
+          mr="40%"
+          adaptiveMxBreakpoint={breakpoint}
+        />
+      );
+
+      await expect(hrComponent(page)).toHaveCSS(
+        "margin-left",
+        `${leftMargin}px`
+      );
+      await expect(hrComponent(page)).toHaveCSS(
+        "margin-right",
+        `${rightMargin}px`
+      );
+    });
+  });
+});
+
+test.describe("Accessibility tests for Hr component", () => {
+  test("should pass accessibility tests for Default story", async ({
```

Comment:
```suggestion
  test("should pass accessibility in it's default composition", async ({
```

or something like that 🤷